docs/overview/changelog.md

```diff
@@ -22,6 +22,7 @@ log][commit-log] for an up-to-date list of all changes.
   to give time to potentially reconfigure which stream to use. You can
   override the delay by running `./manage.py send_zulip_update_announcements --skip-delay`
   once you've done any necessary configuration updates.
+- Removed the Gitter data import tool.
```

Comment:
This doesn't belong here, but in the main changelog, which I should probably do a round of updating.

(This brief list is for things you specifically need to know when upgrading the Zulip server from previous versions, and this detail isn't relevant for upgrades).